build-system-tests/scripts/mega-app-install.sh

```diff
@@ -9,6 +9,7 @@ FRAMEWORK="react"
 FRAMEWORK_VERSION="latest"
 PKG_MANAGER="npm"
 PKG_MANAGER_VERSION="latest"
+TAG="latest"
 
 # Import install function
 source "./scripts/install-with-retries.sh"
```

Comment:
this isn't the install-with-retries script that uses the monorepo, correct?

Reply:
can we rename it to prevent confusion in the future? install-local-deps-with-retries or something?

Reply:
Yeah I renamed it to `install-dependencies-with-retries` since it takes the deps to install as a parameter